### PR TITLE
Feature/jjdd8 ism 6

### PR DESCRIPTION
**Pull request by RobertHilla**
Updated application for filtering thorugh JSON data

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/App.java [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class App {
 
-private static final Logger stdout = LoggerFactory.getLogger("CONSOLE_OUT");
+    private static final Logger stdout = LoggerFactory.getLogger("CONSOLE_OUT");
 
     public static void main(String[] args) {
         stdout.info("Hello World!");
```
Comment (GregChyla) [inline src/main/java/com/infoshareacademy/App.java]: do usunięcia

Comment (GregChyla) [inline src/main/java/com/infoshareacademy/App.java]: do usunięcia

File: src/main/java/com/infoshareacademy/App.java [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
-
-
+        FilterRepository filterRepository = new FilterRepository();
+        filterRepository.initialization();
+        System.out.println(EventsRepository.getEvents());
```
Comment (GregChyla) [inline src/main/java/com/infoshareacademy/App.java]: println cyba też do usunięcia?

File: src/main/java/com/infoshareacademy/service/parser/Parser.java [hunk 3]
```diff
@@ -23,15 +24,8 @@ public void parseData() {
             e.printStackTrace();
         }
         for (Event e : events) {
```
Comment (GregChyla) [inline src/main/java/com/infoshareacademy/service/parser/Parser.java]: ta pętla coś robi?

File: src/main/java/com/infoshareacademy/App.java (hunk 1 above)
Comment (rafalmisiak) [reply src/main/java/com/infoshareacademy/App.java]: +1

File: src/main/java/com/infoshareacademy/App.java (hunk 2 above)
Comment (rafalmisiak) [reply src/main/java/com/infoshareacademy/App.java]: za couty lecą planki, 1 sout - 30 sekund

File: src/main/java/com/infoshareacademy/repository/EventsRepository.java [hunk 4]
```diff
@@ -24,5 +26,9 @@ public static EventsRepository getInstance() {
         return events;
     }
 
+    public static void main(String[] args) {
+        EventsRepository e1 = new EventsRepository();
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/repository/EventsRepository.java]: nazwa a znaczenie

File: src/main/java/com/infoshareacademy/repository/FilterRepository.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
+package com.infoshareacademy.repository;
+
+import com.infoshareacademy.domain.parser.Event;
+import com.infoshareacademy.service.parser.Parser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilterRepository {
+    List<Event> outPutList = new ArrayList<>();
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/repository/FilterRepository.java]: brak przerwy nad zbęda zdublowana pod

File: src/main/java/com/infoshareacademy/repository/FilterRepository.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+
+    public  void initialization(){
+        Parser parser = new Parser();
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/repository/FilterRepository.java]: metoda nazywa sie initialization a parsuje... init niech tworzy instancję parsera a parsuje coś innego

File: src/main/java/com/infoshareacademy/repository/FilterRepository.java [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    public List<Event> allEvents() {
+        for (Event event: EventsRepository.getEvents()) {
+            outPutList.add(event);
+        }
+        return outPutList;
+    }
+
+    public List<Event> filterAllDatesEvents(String date) {
+
+        for (Event event : EventsRepository.getEvents()) {
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/repository/FilterRepository.java]: raz robisz przerwę pod sygnaturą metody a a raz nie

File: src/main/java/com/infoshareacademy/repository/TableRepresentation.java [hunk 8]
```diff
@@ -0,0 +1,5 @@
+package com.infoshareacademy.repository;
+
+public class TableRepresentation {
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/repository/TableRepresentation.java]: klasa widmo?

File: src/main/java/com/infoshareacademy/service/parser/Parser.java (hunk 3 above)
Comment (rafalmisiak) [reply src/main/java/com/infoshareacademy/service/parser/Parser.java]: pętla widmo?

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/domain/parser/Event.java [hunk 9]
```diff
@@ -125,22 +125,22 @@ public void setAttachments(List<Attachments> attachments) {
...
-        string.append("Category ID " + getCategoryId());
-        string.append("start Date " + getStartDate());
-        string.append("Organizer " + getOrganizer());
-        string.append("Active " + getActive());
-        string.append("Desc Short " + getDescShort());
-        string.append("Tickets" + getTickets());
-        return string.toString();
+        StringBuilder eventString = new StringBuilder();
+        eventString.append("Event information");
+        //eventString.append("ID " + getid());
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/domain/parser/Event.java]: nie zostawiamy zakomentowanego kodu w repo

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

File: src/main/java/com/infoshareacademy/repository/FilterRepository.java [hunk 10]
```diff
@@ -0,0 +1,40 @@
+package com.infoshareacademy.repository;
+
+import com.infoshareacademy.domain.parser.Event;
+import com.infoshareacademy.service.parser.Parser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilterRepository {
+    List<Event> outPutList = new ArrayList<>();
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/repository/FilterRepository.java]: nazwa a znaczenie

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**